ArduCopter/Parameters.cpp

```diff
@@ -979,8 +979,8 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
 #if MODE_AUTO_ENABLED == ENABLED
     // @Param: AUTO_OPTIONS
     // @DisplayName: Auto mode options
-    // @Description: A range of options that can be applied to change auto mode behaviour. Allow Arming allows the copter to be armed in Auto. Allow Takeoff Without Raising Throttle allows takeoff without the pilot having to raise the throttle
-    // @Bitmask: 0:Allow Arming,1:Allow Takeoff Without Raising Throttle
+    // @Description: A range of options that can be applied to change auto mode behaviour. Allow Arming allows the copter to be armed in Auto. Allow Takeoff Without Raising Throttle allows takeoff without the pilot having to raise the throttle. Ingore pilot yaw overrides the pilot's yaw stick being used while in auto.
```

Comment:
sorry I didn't catch this earlier but there's a typo in "ignore".

Reply:
the new sentence is a bit odd actually.. I think it has some extra words that it doesn't need.

Reply:
fixed

Reply:
Well fixed the typo, what are the extra words you want out of it?

Reply:
I think, "Ignore pilot yaw overrides **the pilot's yaw stick** being used while in auto" should be, "Ignore pilot's yaw input while in auto"

Reply:
ah, nevermind, i understand now..